mscore/musescore.h

```diff
@@ -219,6 +219,7 @@ class MuseScoreApplication : public QtSingleApplication {
 class MuseScore : public QMainWindow, public MuseScoreCore {
       Q_OBJECT
 
+      friend class Timeline;
```

Comment:
https://github.com/musescore/MuseScore/pull/3274/files#diff-cc622a84f39c126c462e9c5bd168abb6R2791
The editInstrList(), cs, and mixer are private. I can change it if there are underlying issues to using friend, I'll just need to write some access functions